src/Tools/NodeDocumentationMarkdownGenerator/Commands/RenameCommand.cs

```diff
@@ -75,6 +103,29 @@ private static void RenameDirectory(string directory, int maxLength)
             }
 
             var allMdFiles = Directory.GetFiles(directory, "*.md", SearchOption.TopDirectoryOnly).Select(x => new FileInfo(x)).ToList();
+            var allFiles = Directory.GetFiles(directory,"*",SearchOption.TopDirectoryOnly).Select(x => new FileInfo(x)).ToList();
+            var allNonMDFiles = allFiles.Except(allMdFiles);
+            if (Verbose)
+            {
+                //do a scan for support files which do not share the base name.
+                foreach (var nonMDFile in allNonMDFiles)
+                {
+                    foreach (var mdfile in allMdFiles)
+                    {
+                        //md file contains a reference to the support file, but the names are not the same.
```

Comment:
Nice consistency check. I don't remember seeing that case neither. But it was much fewer files when i did this.